src/nomad_material_processing/vapor_deposition/__init__.py

```diff
@@ -62,6 +64,164 @@
 m_package = Package(name='Vapor Deposition')
 
 
+class InsertReduction(ArchiveSection):
+    """
+    The reduction that sometimes is used to lodge the substrate in the substrate holder position..
+    """
+    name = Quantity(
+        type=str,
+        description="""
+        A short and descriptive name for this insert reduction.
+        """,
+        a_eln=ELNAnnotation(
+            component=ELNComponentEnum.StringEditQuantity,
+        ),
+    )
+    insert_id = Quantity(
+        type=str,
+        description="""
+        The ID of the insert that is placed in this position to accomodate the substrate.
+        """,
+        a_eln=ELNAnnotation(
+            component=ELNComponentEnum.StringEditQuantity,
+        ),
+    )
+    material = Quantity(
+        type=str,
+        description="""
+        The material of the insert reduction.
+        """,
+        a_eln=ELNAnnotation(
+            component=ELNComponentEnum.StringEditQuantity,
+        ),
+    )
+    inner_geometry = SubSection(
+        section_def=Geometry,
+    )
+    outer_geometry = SubSection(
+        section_def=Geometry,
+    )
+
+class SubstrateHolderPosition(ArchiveSection):
```

Comment:
Where is the actual position, i.e. x,y relative to center, stored?

Reply:
I added it in the PDi plugin. They use polar coordinates.. i didn't include anyone here. 
It would still be needed to inherit this class to make it Entrydata, so at that point one will add their own positioning parameters

Reply:
added cartesian coords here.